apps/workspaces/triggers.py

```
@@ -1,13 +1,21 @@
+from datetime import datetime
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)